gradle/formatting.gradle

```diff
@@ -6,6 +6,13 @@ allprojects {
 			// non-standard places
 			target '**/*.java'
 
+			importOrder(
```

Comment:
This order was crafted to minimize the impact of the change for existing codebase (so we are down to less than 300 files).